backend/src/utils/session.js

```
@@ -0,0 +1,52 @@
+/**
+ * retrieve clientId from session
+ */
+export function getClientId(session) {
+   try {
+      const clientId = session?.clientId;
+      if (!clientId) {
+         throw new ValidationError("Client ID is required");
+      }
+      return clientId;
+   } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)